src/main/com/fulcrologic/fulcro/algorithms/form_state.cljc

```diff
@@ -597,7 +597,7 @@
   :ret map?)
 
 (defmutation reset-form!
-  "Mutation: Reset the form (recursively) to its (last recorded) pristine state. If form ident is not supplied it uses the ident
+  "Mutation: Reset the form (recursively) to its (last recorded) pristine state. If form ident is not supplied, it uses the ident
```

Comment:
I don't think the comma is necessary.  Why do you think it should have one?  Perhaps I'm mis-remembering sentence rules?

Reply:
Discussed above, will resolve 👍